cpp/include/cudf/utilities/traits.hpp

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include <cudf/lists/list_view.cuh>
+#include <cudf/strings/string_view.hpp>
```

Comment:
Is this necessary? I went through a lot of trouble to remove this include from this file last release.

Reply:
Yeah, it is necessary here because I call `type_dispatcher`, which requires "complete type" for `string_view`.

Reply:
The `type_dispatcher` only needs a forward reference to `string_view` which this file already has.
It may be because you are doing an arbitrary `sizeof` on all non-fixed-point types.
Could you just SFNIAE out `string_view` from logical-cast altogether?